### ISSUE TITLE
helper text crowd_values

### ISSUE THREAD

**Issue by williamhoos** [clinwiki-org/clinwiki#1059]
Should be done only after helper text for crowd_keys are complete

Written assuming mailmerge/handlebar support

- [ ] Add crowd_value_helper_text, crowd_value_description, crowd_value_default_icon
- [ ] Add new field support to island_config for "bucketTemplate"
- [ ] Support current Bucket Format which is "crowd_value (docCount)" see below for possible format support
- [ ] consider supporting agg facets that could get formatting from crowd_key and crowd_value table (needs scoping)
- [ ] other?

New format of helper text with and without document count in CHECKBOX bucket list, creating new display name, maintaining current value for "on submit"
"bucketTemplate" : "{{crowd_value}} ({{docCount}})"

Alternative template examples enabled:
No counts
"bucketTemplate" : "{{crowd_value}}"

Reverse order, no parenthesis
"bucketTemplate" : "{{docCount}} {{crowd_value}}"

Adding helper text
"bucketTemplate" : "{{crowd_value}} ({{docCount}}) [span class="mm-tooltip"][u][i class="fa fa-info-circle"][/u][span class="mm-tooltiptext mm-tooltip-tr"]{{crowd_value_helper_text}}[/span]" (note: using ] and [ to avoid html in issue comments here from > and < "

Would also support using crowd_value_description instead of crowd_value for display of bucketTemplate

mock-up
![image](https://user-images.githubusercontent.com/19411659/125231435-b7be0980-e2a8-11eb-9050-4d2410da29f1.png)

**Comment by williamhoos** [comment]
the example mockup above is actually a ctgov field, not a crowd_key, so need to discuss basic support for non-crowd aggs. Should work to still set-up as a crowd_key and crowd_values, but need to confirm how the info will be pulled from combination of agg query to elastic and list of crowd_values (ie how a "missing" crowd_value table entry that is present in the ctgov data (and therefore the index) is merged into the list of values in the facet without error
